lua/lspconfig.lua

```diff
@@ -78,10 +92,12 @@ end
 
 local minimum_neovim_version = '0.9'
 if vim.fn.has('nvim-' .. minimum_neovim_version) == 0 then
+  local minimum_vim_version = vim.version.parse(minimum_neovim_version)
```

Comment:
`vim.version.parse` was not available in older versions, that's why it was intentionally not used.

Reply:
oh interesting, `vim.version.parse` was used. I was just moving it. How would you like to resolve it?

Reply:
Ok, I moved it to just use the string rather than using `vim.version.parse` like the code was using originally